2021-06-14 20:06:59,551 ERROR: Exception on /matches [GET] [in /home/katze/.local/lib/python3.8/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/katze/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/katze/.local/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/katze/.local/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/katze/.local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/katze/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/katze/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/katze/.local/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/katze/Documents/projects/bgl/app/routes.py", line 239, in matches
    return render_template('matches.html', title='Matches', matches=matches, list=list)
  File "/home/katze/.local/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/katze/.local/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/katze/.local/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/katze/.local/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/katze/.local/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/katze/Documents/projects/bgl/app/templates/matches.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/katze/Documents/projects/bgl/app/templates/base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "/home/katze/.local/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/home/katze/.local/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/home/katze/.local/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/home/katze/.local/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in block "body"
    {% block content -%}
  File "/home/katze/Documents/projects/bgl/app/templates/matches.html", line 5, in block "content"
    {% include '_match.html' %}
  File "/home/katze/Documents/projects/bgl/app/templates/_match.html", line 16, in top-level template code
    <td>{{ ' ,'.join( match.stats | map(attribute='player') | list  ) }}</td>
TypeError: sequence item 0: expected str instance, Undefined found
2021-06-14 20:07:20,729 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:07:27,376 ERROR: Exception on /matches [GET] [in /home/katze/.local/lib/python3.8/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/katze/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/katze/.local/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/katze/.local/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/katze/.local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/katze/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/katze/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/katze/.local/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/katze/Documents/projects/bgl/app/routes.py", line 239, in matches
    return render_template('matches.html', title='Matches', matches=matches, list=list)
  File "/home/katze/.local/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/katze/.local/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/katze/.local/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/katze/.local/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/katze/.local/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/katze/Documents/projects/bgl/app/templates/matches.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/katze/Documents/projects/bgl/app/templates/base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "/home/katze/.local/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/home/katze/.local/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/home/katze/.local/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/home/katze/.local/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in block "body"
    {% block content -%}
  File "/home/katze/Documents/projects/bgl/app/templates/matches.html", line 5, in block "content"
    {% include '_match.html' %}
  File "/home/katze/Documents/projects/bgl/app/templates/_match.html", line 16, in top-level template code
    <td>{{ ' ,'.join( match.stats | map(attribute='player')) }}</td>
TypeError: sequence item 0: expected str instance, Undefined found
2021-06-14 20:07:42,208 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:07:51,709 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:08:36,735 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:08:44,870 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:10:11,439 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:10:49,933 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:13:00,280 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:15:19,793 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:16:17,528 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:16:36,921 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:16:55,601 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:17:31,070 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:18:18,925 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:18:53,596 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:19:25,739 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:20:22,802 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:20:55,490 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:22:12,316 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:23:11,357 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:23:50,587 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
2021-06-14 20:24:13,641 INFO: BGL Stats [in /home/katze/Documents/projects/bgl/app/__init__.py:34]
